README.md

```diff
@@ -70,19 +71,19 @@ _We are discussing the naming and API for this functionality in [#25](https://gi
 
 ### `childKey` property
 
-Type: `function(item: any) => any`
+Type: `function(itemIndex: number) => any`
 
 Set this property to provide a custom identifier for the element corresponding to a given item.
 
 This is often used for more efficient re-ordering, as seen in [the example below](#efficient-re-ordering-using-childkey).
 
 ### `items` property

```

Comment:
We might change this soon anyway, but I'd find `totalItems` to be a better name probably.